### ISSUE TITLE
npm run build fails.

### ISSUE THREAD

**Issue by ovenwerks** [jean-emmanuel/open-stage-control#17]
npm start works fine but npm run build does not.

Included npm-debug.log
[npm-debug.log.zip](https://github.com/jean-emmanuel/open-stage-control/files/224700/npm-debug.log.zip)

**Comment by jean-emmanuel** [comment]
You need Node.js 4.0 or higher to build the app (you have 0.10.25 
according to the log). I'm adding this to the docs right away.

**Comment by ovenwerks** [comment]
On Mon, 18 Apr 2016, jean-emmanuel wrote:

> You need Node.js 4.0 or higher to build the app (you have 0.10.25
> according to the log). I'm adding this to the docs right away.

Ga, It will have to wait till I go from LTS to LTS unless I can build from 
source. Ubuntu does go up to 4.* untill 16.04 (releases later this week.) 
I am not going to install a random binary at this point :)

--
Len Ovens
www.ovenwerks.net

**Comment by jean-emmanuel** [comment]
Here is a workaround :
- open `package.json`
- replace `"electron-packager": "7.0.0"` with `"electron-packager": "6.0.2"`
Support for early versions of nodejs was dropped in the latest version 
of this dependency but the previous version should work fine.

**Comment by ovenwerks** [comment]
On Mon, 18 Apr 2016, jean-emmanuel wrote:

> Here is a workaround :
> - open `package.json`
> - replace `"electron-packager": "7.0.0"` with `"electron-packager": "6.0.2"`
> Support for early versions of nodejs was dropped in the latest version
> of this dependency but the previous version should work fine.

Thank you. I actually found that I was able to download the source and 
build. Both on the i5 and the RPi. Electron packager has a newer version 
out that should work for arm, but I may wait on that till I actually need 
to work on the Pi for HW sake. (buttons I can use now, but encoders need 
HW support)

The compact faders are wonderful, BTW. The Pre-args make strip detection a 
breeze too. (I am now working on middleware)


--
Len Ovens
www.ovenwerks.net
